ansible/playbooks/sap-hana-preconfigure.yaml

```diff
@@ -36,7 +43,7 @@
         state: present
         replacefiles: true
       when:
-        - ansible_facts['distribution_version'] == '15.1'
+        - ansible_facts['distribution_version'] == '15.1' and not use_hana_sr_angi | bool
```

Comment:
Do we plan to have Angi ported to 15.1? If not we can remove this `and` and add an assert task like: 
```
died if use_hana_sr_angi and ansible_facts['distribution_version'] == '15.1'
```

Reply:
It's just sanitation.